follower_control:
  cmd_vel_topic: '/cmd_vel'
  target_topic: '/follower/target'
  twist_mul: 1.0 # Multiplier for target to twist conversion (m to m/s conversion)